pygfunction/api.py

```diff
@@ -0,0 +1,120 @@
+from pygfunction.gfunction import gFunction
+from pygfunction.boreholes import Borehole
+
+from typing import Union, List, Tuple, Dict, Iterable
+
+FloatOrList = Union[float, List[float]]
+
+
+class BoreholeFieldParameters:
+    """This class represents the borehole field and can generate inputs for pygfunction"""
+
+    def __init__(self):
+        self.height_list: List[float] = []  # TODO: Document the units
+        self.depth_list: List[float] = []
+        self.bh_radius_list: List[float] = []
+        self.tilt_angle_list: List[float] = []  # TODO: Is this degrees? And I assume from vertical?
+        self.orientation_angle_list: List[float] = []  # TODO: Same question
+        self.x_coords: List[float] = []  # TODO: What are the units here, I'm assuming meters
+        self.y_coords: List[float] = []  # TODO: Same question
```

Comment:
OK, modified this class just a bit.  Now the constructor simply instantiates the lists.  This will allow for us to create any variety of setup methods later on that can flexibly build out borehole fields, and the constructor won't need to change.